.env

```
@@ -233,6 +233,7 @@ VITE_STORY_NODE_URL=https://mainnet.storyrpc.io
 VITE_SUI_NODE_URL=https://fullnode.mainnet.sui.io:443
 VITE_TON_NODE_URL=https://toncenter.com/api/v2/jsonRPC
 VITE_TRON_NODE_URL=https://api.trongrid.io
+VITE_TRON_GRID_API_KEY=17430894-392e-44e8-b015-4a9c4fe17546
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_TRON_GRID_API_KEY key should go before the VITE_TRON_NODE_URL key

(UnorderedKey)